app/lib/three_scale/redis_config.rb

```diff
@@ -56,5 +58,16 @@ def parse_sentinels(sentinels)
         parsed_sentinel
       end
     end
+
+    def load_default_ca_cert(conf)
+      return if conf.dig(:ssl_params, :ca_file).present? || conf.dig(:ssl_params, :ca_path).present?
+
+      cert_path = Rails.root.join('config/ca_cert.pem').to_s
```

Comment:
I forgot if we discussed this already. But shouldn't default name be redis specific? This config foleder seems to be global for porta. But this is redis specific config. Maybe `redis_ca.pem`?

btw `ca_file` is more correct than `ca_cert` because it can be a single cert, a cert chain and I believe a whole bunch of CA certs. If the idea is that this fine be a global CA file for porta, then also better to rename.

Reply:
Yes, you mentioned that [here](https://github.com/3scale/porta/pull/3572/commits/1f518574c054da0a41725d507b7e5407a1ecc69c#r1397103552)

I don't think `ca_cert.pem` is so bad name. `cert.pem` t's the default name for the system certificate bundle in Debian, and also in RHEL for compatibility with Debian. Check your `/etc/ssl` folder.